algorithms/Java/sorting/cyclic-sort.java

```diff
@@ -0,0 +1,71 @@
+package CyclicSort;
+import java.util.Arrays;
+
+public class cyclic-sort {
+    public static void main(String[] args)
+    {
+        int[] array1 = {4,5,2,3};
+        int[] array2 = {2, 1, 5, 2, 9, 1, 2, 3, 3, 4, 4, 5, 6, 7, 8};
+
+        cyclic sorter = new cyclic();
+        sorter.cycleSort(array1);
+        sorter.cycleSort(array2);
```

Comment:
Thanks, @Ashwin-exe for your effort,

Please try to fix line 10, where is the class called `cyclic`? 👇
Main.java:9: error: cannot find symbol
        cyclic sorter = new cyclic();